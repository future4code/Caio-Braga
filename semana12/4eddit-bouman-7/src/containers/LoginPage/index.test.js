import React from 'react';
import renderer from 'react-test-renderer';


test('Snapshot', () => {
    const tree = renderer.create(<handleLoginButton />).toJSON();
    expect(tree).toMatchSnapshot();
})
