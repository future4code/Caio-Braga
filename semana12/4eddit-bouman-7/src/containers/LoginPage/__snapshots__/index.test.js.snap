// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot 1`] = `<handleLoginButton />`;

exports[`Snapshot 2`] = `<handleLoginButton />`;
